### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by RealAnishSharma**
Certain typo changes.

### CODE REVIEW

File: C++/README.md [hunk 1]
```diff
@@ -58,7 +58,8 @@ To remove all of the files created during compilation, run **`make clean`**. You
...
 
 ## Unit tests
-If you would like to (and you should) add unit tests for the code you are contributing, or for some existing code, read about [Adding unit tests for C++ code](UNIT_TESTS.md).
+If you would like to (and you should) add unit tests for the code you are contributing, or for some of the existing code, read about [Adding unit tests for C++ code](UNIT_TESTS.md).
+
 
 ## C++ Coding Guidelines
-If you want to contribute an algorithm or data structure in C++, make sure to read the [C++ Coding Guidelines](CODING_GUIDELINES.md) apart from the general [Contributing Guidelines](../CONTRIBUTING.md). This will help in making your code conform to the style followed in this project.
+If you want to contribute an algorithm or data structure in C++ (new and innovative), make sure to read the [C++ Coding Guidelines](CODING_GUIDELINES.md) apart from the general [Contributing Guidelines](../CONTRIBUTING.md). This will help in ensuring that your code conforms to the style followed in this project.
```
Comment (faheel) [inline C++/README.md]: "(new and innovative)" can be removed. The other changes are good :+1:

File: C++/README.md [hunk 2]
```diff
@@ -58,7 +58,8 @@ To remove all of the files created during compilation, run **`make clean`**. You
 To know what happens in the background when **`make`** runs, you may read the [Makefile](Makefile). For more information on **`make`**, read the [GNU make Manual](https://www.gnu.org/software/make/manual/make.html).
 
 ## Unit tests
-If you would like to (and you should) add unit tests for the code you are contributing, or for some existing code, read about [Adding unit tests for C++ code](UNIT_TESTS.md).
+If you would like to (and you should) add unit tests for the code you are contributing, or for some of the existing code, read about [Adding unit tests for C++ code](UNIT_TESTS.md).
+
```
Comment (faheel) [inline C++/README.md]: Please remove the extra blank line (line 62).

File: C++/README.md [hunk 3]
```diff
@@ -37,7 +37,7 @@ Implementation of well-known (and some rare) algorithms, in C++.
   * [Shell sort](source/sorting/shell_sort.cpp)
 * String
   * [Longest common subsequence](source/string/lcs.cpp)
-  * Searching
+  * Searching [search a pattern within string]
```
Comment (faheel) [inline C++/README.md]: You can change the text inside the brackets to "pattern matching". Also, please change the square brackets to parenthesis :grimacing:

**Review by faheel — CHANGES_REQUESTED** [review]

File: C++/README.md (hunk 3 above)
Comment (RealAnishSharma) [reply C++/README.md]: sure sure

**Comment by RealAnishSharma** [comment]
Hope its better now... i wish i was productive here, though quite meagerly.

**Comment by faheel** [comment]
Definitely! Every bit counts. Thanks a lot for your contribution :smiley:

**Comment by RealAnishSharma** [comment]
indeed ... Thanks a lot for giving me the opportunity 😸 :

**Pull request merged**